api/handlers.go

```diff
@@ -15,7 +15,7 @@ import (
 type GetVarsFunc func(r *http.Request) map[string]string
 
 // CreateSessionHandlerFunc returns a function that generates a session. Method = "POST"
-func CreateSessionHandlerFunc(sessions Sessions, cache Cache) http.HandlerFunc {
+func CreateSessionHandlerFunc(sess Session, cache Cache) http.HandlerFunc {
```

Comment:
I don't personally like partial variable names like `sess`, I can see sense in short vars like `s` if the code is trivial enough or the variable is used frequently but personally I would expand this one to the full word. I'm not sure what our standards are on this though.